tests/pr_test.go

```diff
@@ -48,3 +68,23 @@ func TestRunCompleteExample(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
```

Comment:
You can remove the `TestRunCompleteExample` now that we have the FSCloud example test. The complete example will still get tested as part of the upgrade test

Reply:
Done

Reply:
I have added back the complete example as upgrade example will be skipped due to CBR.

Reply:
I assume this is just for testing to ensure complete examples still works after removing CBR from it? The code we merge here will not have a complete example test in it. It will have FSCloudComplete and CompleteUpgrade